admin/services/pin-generation.service.js

```diff
@@ -12,11 +12,16 @@ const pupilAttendanceService = require('../services/attendance.service')
 const pupilIdentificationFlagService = require('../services/pupil-identification-flag.service')
 const restartService = require('../services/restart.service')
 const config = require('../config')
+const logger = require('./log.service').getLogger()
 
 const allowedWords = new Set(
   (config.Data.allowedWords && config.Data.allowedWords.split(',')) || []
 )
 
+const bannedWords = [
```

Comment:
@jon-shipley can't we just remove the word from the env variable? 🤔

Reply:
It's a safeguard from incorrect config